lib/apptimize_flutter.dart

```diff
@@ -220,23 +244,32 @@ class Apptimize {
   /// variant this user/device is enrolled in.
   static Future<void> runTest(
       String testName, Function baseline, Map<String, Function> codeblocks,
-      [int updateMetadataTimeout]) async {
-    final String codeblock = await _channel.invokeMethod('runTest', {
+      [int? updateMetadataTimeout]) async {
+    final String? codeblock = await _channel.invokeMethod('runTest', {
       "testName": testName,
       "codeBlocks": codeblocks.keys.toList(),
       "updateMetadataTimeout": updateMetadataTimeout
     });
 
-    Function block = codeblocks[codeblock] ?? baseline;
-    block();
+    if (codeblock != null) {
+      Function block = codeblocks[codeblock] ?? baseline;
+      block();
+    } else {
+      baseline();
```

Comment:
Yes, it is legal. The native side returns `null` when the baseline is executed. What IS unlikely is for `codeblock` to be a string that is not contained in the `codeblocks` map. I've changed this.

It will still always at _least_ run the baseline (as specified in the comments).